File: Projects/Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/OrderManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface IOrderManager : IManagerBase<Order, int>
    {
	}
	
	partial class OrderManager : ManagerBase<Order, int>, IOrderManager
    {
	}
}

File: Projects/Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/OrderManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface IOrderManager
	'	Inherits IManagerBase(Of Order, System.Int32)
	'End Interface

	Partial Class OrderManager
		Inherits ManagerBase(Of Order, System.Int32)
		Implements IOrderManager
	End Class
End Namespace
